src/assessments/native-widgets/test-steps/instructions.tsx

```diff
@@ -1,44 +1,49 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import * as React from 'react';
-
 import { DefaultWidgetPropertyBag } from 'common/types/property-bag/default-widget';
 import { VisualizationType } from 'common/types/visualization-type';
 import { link } from 'content/link';
-import { productName } from 'content/strings/application';
-import { TestAutomaticallyPassedNotice } from 'content/test/common/test-automatically-passed-notice';
 import * as content from 'content/test/native-widgets/instructions';
 import { AssessmentVisualizationEnabledToggle } from 'DetailsView/components/assessment-visualization-enabled-toggle';
 import { ScannerUtils } from 'injected/scanner-utils';
+import * as React from 'react';
+
 import { AnalyzerConfigurationFactory } from '../../common/analyzer-configuration-factory';
 import { AssistedTestRecordYourResults } from '../../common/assisted-test-record-your-results';
+import { InstructionsAndLabelsNotes } from '../../common/instructions-and-labels-note';
 import { NoValue, PropertyBagColumnRendererConfig } from '../../common/property-bag-column-renderer';
 import { PropertyBagColumnRendererFactory } from '../../common/property-bag-column-renderer-factory';
 import * as Markup from '../../markup';
 import { ReportInstanceField } from '../../types/report-instance-field';
 import { Requirement } from '../../types/requirement';
 import { NativeWidgetsTestStep } from './test-steps';
 
-const description: JSX.Element = <span>If a native widget has visible instructions, they must be programmatically related to it.</span>;
+const description: JSX.Element = (
+    <span>If a native widget has a visible label or instructions, they must be programmatically related.</span>
+);
 
 const howToTest: JSX.Element = (
     <div>
         <p>
-            For this requirement, {productName} highlights native widgets. Native widgets include
+            The visual helper for this requirement highlights native widgets. Native widgets include
             <Markup.NonBreakingSpace />
-            <Markup.Tag tagName="button" isBold={true} />,
+            <Markup.Tag tagName="button" isBold={false} />,
```

Comment:
Just wondering, does Markup.Tag default to bold?

Reply:
yes... I remove the `isBold` first, and found out the tags we're still bold. I check the code and yes, we are explicitly defaulting to boldness on `Markup.Tag`